src/main/scala/lms/thirdparty/mpi.scala

```diff
@@ -0,0 +1,83 @@
+package lms.thirdparty
+
+import lms.core._
+import lms.util._
+import lms.core.stub._
+import lms.core.Backend._
+import lms.core.virtualize
+import lms.core.utils.time
+import lms.macros.SourceContext
+
+import lms.collection._
+trait MPIOps { b: Base with PointerOps =>
```

Comment:
Any reason why you don't just extend the trait instead of using that?

Reply:
no reason :) Will do extends :)